terraform/eks/daemon/gpu/main.tf

```diff
@@ -306,7 +306,7 @@ resource "kubernetes_daemonset" "exporter" {
+            "/bin/echo 'DCGM_FI_DEV_GPU_UTIL{PodName=\"gpu-burn\",gpu=\"0\",UUID=\"uuid0\",device=\"nvidia0\",modelName=\"Tesla T4\",Hostname=\"hostname1\",container=\"main\",namespace=\"amazon-cloudwatch\",pod=\"gpu-burn-0\"} 1\nDCGM_FI_DEV_FB_FREE{PodName=\"gpu-burn\",gpu=\"0\",UUID=\"uuid0\",device=\"nvidia0\",modelName=\"Tesla T4\",Hostname=\"hostname1\",container=\"main\",namespace=\"amazon-cloudwatch\",pod=\"gpu-burn-0\"} 1\nDCGM_FI_DEV_FB_USED{PodName=\"gpu-burn\",gpu=\"0\",UUID=\"uuid0\",device=\"nvidia0\",modelName=\"Tesla T4\",Hostname=\"hostname1\",container=\"main\",namespace=\"amazon-cloudwatch\",pod=\"gpu-burn-0\"} 1\nDCGM_FI_DEV_FB_TOTAL{PodName=\"gpu-burn\",gpu=\"0\",UUID=\"uuid0\",device=\"nvidia0\",modelName=\"Tesla T4\",Hostname=\"hostname1\",container=\"main\",namespace=\"amazon-cloudwatch\",pod=\"gpu-burn-0\"} 1\nDCGM_FI_DEV_FB_USED_PERCENT{PodName=\"gpu-burn\",gpu=\"0\",UUID=\"uuid0\",device=\"nvidia0\",modelName=\"Tesla T4\",Hostname=\"hostname1\",container=\"main\",namespace=\"amazon-cloudwatch\",pod=\"gpu-burn-0\"} 1\nDCGM_FI_DEV_GPU_TEMP{PodName=\"gpu-burn\",gpu=\"0\",UUID=\"uuid0\",device=\"nvidia0\",modelName=\"Tesla T4\",Hostname=\"hostname1\",container=\"main\",namespace=\"amazon-cloudwatch\",pod=\"gpu-burn-0\"} 1\nDCGM_FI_DEV_POWER_USAGE{PodName=\"gpu-burn\",gpu=\"0\",UUID=\"uuid0\",device=\"nvidia0\",modelName=\"Tesla T4\",Hostname=\"hostname1\",container=\"main\",namespace=\"amazon-cloudwatch\",pod=\"gpu-burn-0\"} 1\nDCGM_FI_PROF_PIPE_TENSOR_ACTIVE{PodName=\"gpu-burn\",gpu=\"0\",UUID=\"uuid0\",device=\"nvidia0\",modelName=\"Tesla T4\",Hostname=\"hostname1\",container=\"main\",namespace=\"amazon-cloudwatch\",pod=\"gpu-burn-0\"} 1' >> /usr/local/apache2/htdocs/metrics && sed -i -e \"s/hostname1/$HOST_NAME/g\" /usr/local/apache2/htdocs/metrics && httpd-foreground -k restart"
```

Comment:
Main change is the PodName and pod labels to match our gpu-burn statefulset and service.